packages/villain-react/dev-sandbox/src/components/field.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import clsx from 'clsx'
+//import Villain from '@/index.js'
```

Comment:
`@` is just an alias of `villian-react/src`, this wont work inside the cra, you should probably use `../` until you find the right path

Reply:
This is what you need to import on that line:
`/packages/villain-react/src/index.js`

Reply:
relative path imports for files outside the src folder in cra are not supported.
I tried doing `lerna add villain-react --scope=dev-sandbox` (sibling dependency) but it prompted `lerna: command not found`. I have everything installed, lerna logs out when I run the script: `yarn start:react-new`.
Don't know what's going on...

Reply:
Hmm I didn't know about this restriction, I did a quick search and it looks like there are a bunch of hacks but requires to modify the configuration or use external modules.
https://github.com/timarney/react-app-rewired

Reply:
I'm not sure how to fix this either, I'll need more time to look in to it.

Reply:
https://github.com/transitive-bullshit/create-react-library

Reply:
This is the best of option that I can think for now ^^

Reply:
Sorry for the trouble, makes no sense to me why cra blocks imports outside `./src/`

Reply:
No issues haha.
I try using `yarn workspace` commands and see if we can get around this.

Reply:
I tried linking villain-web into dev-sandbox using yarn link but it didn't work. I had to add babel-loader as a dependency to villain-web for building it. That went well. But I guess the `yarn link` command is broken. The issue here is something related to `~/.config` being a symlink instead of a folder on my computer.
I am not that comfortable with Linux and I might have to investigate this further before making some changes on my OS. This might take a while.